src/date/expand-pattern.js

```diff
@@ -1,3 +1,24 @@
+function getDateTime( type, cldr ) {
+	var result;
+	result = cldr.main([
+		"dates/calendars/gregorian/dateTimeFormats",
+		type
+	]);
+	if ( result ) {
+		result = formatMessage( result, [
+			cldr.main([
+				"dates/calendars/gregorian/timeFormats",
+				type
+			]),
+			cldr.main([
+				"dates/calendars/gregorian/dateFormats",
+				type
+			])
+		]);
+	}
+	return result;
+}
+
```

Comment:
@rxaviers sir,

Sir, there is difference in path used in `cldr.main` of blocks of code mentioned. If wanted, we can add a `if` block for `case "datetime" in pattern:` and check for `"datetime" in pattern:` or `"skeleton" in pattern:` to handle both in one function. Should I add?